### PR TITLE
Update header on data page

### PR DESCRIPTION
**Pull request by dmantonyuk**
Add same header to advocacy.code.org/stateofcs/data as advocacy.code.org/stateofcs

## Links

https://codedotorg.atlassian.net/browse/ACQ-1133

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Comment by dmantonyuk** [comment]
download

File: pegasus/sites.v3/advocacy.code.org/views/mobile_footer_responsive.haml
Comment (kelbyhawn) [inline pegasus/sites.v3/advocacy.code.org/views/mobile_footer_responsive.haml:1]: Does the footer need to be updated here too?

Comment (kelbyhawn) [inline pegasus/sites.v3/advocacy.code.org/views/mobile_footer_responsive.haml:1]: Let's keep this footer as it was for now and it can be redesigned later.

**Review by kelbyhawn — COMMENTED** [review]
Looks good! One comment about putting the original `views/mobile_footer_responsive.haml` view back, but once that's done it's good to go.

**Review by kelbyhawn — APPROVED** [review]

**Pull request merged**